nsv13/code/modules/vehicles/_vehicle.dm

```diff
@@ -177,7 +179,7 @@ MASSIVE THANKS TO MONSTER860 FOR HELP WITH THIS. HE EXPLAINED PHYSICS AND MATH T
 			if(occupants.len)
 				var/mob/M = return_drivers()[1]
 				var/client/C = M.client
-				if(C && C.keys_held["Alt"])
+				if(!M || C && C.keys_held["Alt"]) //No driver? Brake automatically. Otherwise check if driver wants to break.
```

Comment:
does this need to process once nobody is in it and its not moving?
Also I believe it needs a extra set of ( ) somewhere in here.